index.html

```diff
@@ -63,15 +62,18 @@ <h1>Popover Attribute Polyfill</h1>
                 <div>
                   <div id="shadowedNestedPopover" popover="auto">Shadowed Nested Popover (auto)</div>
                 </div>`;
-        shadowRoot.adoptedStyleSheets = [sheet];
+        const link = document.createElement('link');
+        link.rel = 'stylesheet';
+        link.href = './dist/popover.css';
+        shadowRoot.prepend(link);
         document.getElementById('crossTreeToggle').popoverTargetElement =
           shadowRoot.getElementById('shadowedPopover');
       </script>
     </div>
 
     <div id="buttons">
       <button popovertarget="popover1" aria-expanded="false">
-        Click to toggle Popover 1
+        <span>Click to toggle Popover 1</span>
```

Comment:
With this change on `main` I am able to reproduce the bug, and confirm that it is fixed in this PR.